Makefile

```diff
@@ -222,6 +222,10 @@ ifneq ( ,$(findstring image_mirror_mapping,$(IMAGE_MAPPINGS)))
 	@ oc apply -f integreatly-delorean-secret.yml --namespace=$(NAMESPACE)
 endif
 
+.PHONY: cluster/prepare/delorean/pullsecret
```

Comment:
@StevenTobin wdy think about putting this into `cluster/prepare/delorean` make target above. It has the check for whether it's a delorean pr included?

Reply:
I don't know if we want to do both, since the makefile change you introduced has less impact on the cluster and the one I'm introducing here will cause every node to drain and reboot so for products where we have a pullsecret solution it's much less impact to just use cluster/prepare/delorean.

Reply:
We do not want this to happen all the time, but ideally if it needs to be done it should just happen as part of the `cluster/prepare/delorean` target. I guess for that we would need some way of knowing if a manifest existed in the manifests dir that required this to be done :thinking: